spec/order_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'order'
+
+describe Order do
+
+  subject(:order) { described_class.new }
+
+  let(:menu) { double(:menu, :dishes => { 'Spring Roll' => 0.99 }) }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)